### PR TITLE
Photos Team 

### PR DESCRIPTION
**Pull request by FreddieCodes**
- user can upload photos on page 
- uploaded photos are added to database 
- photos pages UI's
- uploaded photos can be seen on page 

*ISSUE: 
- dependency on "image magick" which is not a Gem, advise to integrate image magick installation to heroku. 

Use:
- visit /photos to begin

### CODE REVIEW

File: app/assets/stylesheets/photos.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Photos controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (neoeno) [inline app/assets/stylesheets/photos.scss]: We can get rid of these generated JS/Stylesheet files

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :photos
```
Comment (neoeno) [inline config/routes.rb]: This should probably go after the explanatory comment — otherwise it'll just end up randomly in the middle.

File: public/404.html [hunk 3]
```diff
@@ -1,67 +0,0 @@
...
-  <!-- This file lives in public/404.html -->
-  <div class="dialog">
-    <div>
-      <h1>The page you were looking for doesn't exist.</h1>
-      <p>You may have mistyped the address or the page may have moved.</p>
-    </div>
-    <p>If you are the application owner check the logs for more information.</p>
-  </div>
-</body>
-</html>
```
Comment (neoeno) [inline public/404.html:67]: Why are these deleted?

File: spec/models/photo_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Photo, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (neoeno) [inline spec/models/photo_spec.rb]: Do what it says!

File: spec/helpers/photos_helper_spec.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+# describe PhotosHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe PhotosHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (neoeno) [inline spec/helpers/photos_helper_spec.rb]: Do what it says!

File: spec/requests/photos_spec.rb [hunk 6]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "Photos", type: :request do
+  describe "GET /photos" do
+    it "works! (now write some real specs)" do
```
Comment (neoeno) [inline spec/requests/photos_spec.rb]: Delete or update

File: test/system/photos_test.rb [hunk 7]
```diff
@@ -0,0 +1,9 @@
+require "application_system_test_case"
+
+class PhotosTest < ApplicationSystemTestCase
+  # test "visiting the index" do
+  #   visit photos_url
+  #
+  #   assert_selector "h1", text: "Photo"
+  # end
```
Comment (neoeno) [inline test/system/photos_test.rb]: Commented out code. I'd remove this file

**Review by neoeno — DISMISSED** [review]
There's a lot of scaffolded code here. Scaffolds and generators are a start, but not a finish. I'd go through this diff, figure out what you need and what you don't, and refine it down.

**Review by gsgkw — APPROVED** [review]
looks fine

**Comment by nick-otter** [comment]
Heroku DB migrate required, see db/schema.rb changes

**Review by nick-otter — APPROVED** [review]

File: public/404.html (hunk 3 above)
Comment (nick-otter) [reply public/404.html:67]: Deleted public folder to ensure no conflicts with master
